InternetHospital.WebApi/Controllers/FeedBackController.cs

```diff
@@ -32,26 +30,33 @@ public class FeedBackController : ControllerBase
         [HttpPost("Create")]
         public IActionResult CreateFeedBack([FromBody]FeedBackCreationModel feedBackCreationModel)
         {
-
-            int userId;
-            Int32.TryParse(User.Identity.Name, out userId);
-
-            if (feedBackCreationModel != null)
+            const int MIN_TEXT_LENGTH = 10;
+            const int WRONG_TYPE_ID = 0;
+            if (int.TryParse(User.Identity.Name, out int userId))
             {
-                if (feedBackCreationModel.Text.Length <= 10)
-                    return NotFound(new { message = "The text not in correct format" });
-                if (feedBackCreationModel.TypeId == 0)
-                    return NotFound(new { message = "Feedback format is invalid" });
+                feedBackCreationModel.UserId = userId;
+
+                if (feedBackCreationModel != null)
+                {
+                    if (feedBackCreationModel.Text.Length <= MIN_TEXT_LENGTH)
+                        return NotFound(new { message = "The text not in correct format" });
+                    if (feedBackCreationModel.TypeId == WRONG_TYPE_ID)
+                        return NotFound(new { message = "Feedback format is invalid" });
+                    else
+                    {
+                        _feedBackService.FeedBackCreate(feedBackCreationModel);
```

Comment:
this method returns value. Change signature to return db save result(success or not) and return it from back-end to UI